src/azure-cli/azure/cli/command_modules/acr/check_health.py

```diff
@@ -72,11 +72,60 @@ def _subprocess_communicate(command_parts, shell=False):
     return output, warning, stderr, succeeded
 
 
+# Retrieve blob identified by digest - GET {url}/v2/{name}/blobs/{digest}
+def _get_blob(login_server, repository_name, digest):
```

Comment:
is this API only being used in check health? It feels it might be used by multiple files if so let's have in one place

Reply:
It's not being used in other files, but I could put it in '._docker_utils.py' so it could be found and re-used more easily.

Reply:
nah it's ok. Whoever wants to reuse it can do that change.